app/assets/stylesheets/blacklight/_header.scss

```diff
@@ -53,3 +53,31 @@
     }
   }
 }
+
+#skip-link {
+  position: absolute;
+  top: 5px;
+  left: 10px;
+  z-index: 1;
+
+  .element-invisible {
+    position: absolute !important;
+    clip: rect(1px, 1px, 1px, 1px);
+    overflow: hidden;
+    height: 1px;
+  }
+
+  .element-invisible.element-focusable:active,
+  .element-invisible.element-focusable:focus {
+    position: static !important;
+    clip: auto;
+    overflow: visible;
+    height: auto;
+  }
+
+  a:focus {
+    background-color: white;
+    border-radius: 0 0 0.2rem 0.2rem;
+    padding: .5rem 1rem;
```

Comment:
I believe we can take advantage of some Bootstrap utility classes for [border radius](https://getbootstrap.com/docs/4.0/utilities/borders/#border-radius) and [spacing](https://getbootstrap.com/docs/4.3/utilities/spacing/#notation) instead of hardcoding them here.

Reply:
Is it possible to use those Bootstrap classes in the scss file, or do I just add them to each skip link in the views?

Reply:
Eh, I just added them to each skip link. There's only three right now.